content/donate.adoc

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: "Donate to Jenkins"
+---
+
+NOTE: You can donate directly through the
+link:https://co.clickandpledge.com/advanced/default.aspx?wid=46160[SPI donation
+page] (donation in US dollars) or via the
+link:http://www.ffis.de/Verein/donations.html[ffis.de donation page] (donation
+in Euros) if you do not need the context below
+
+
+
+The Jenkins project is affiliated with link:http://www.spi-inc.org[Software in
+the Public Interest], a non-profit organization. Via SPI we can accept
+link:https://co.clickandpledge.com/advanced/default.aspx?wid=46160[donations
+online] in US dollars. For other ways to donate, refer to the
+link:http://spi-inc.org/donations[donations page on the SPI website].
+
+
+For European residents, we can also accept donations through
```

Comment:
Reads like a wiki page (i.e. a later addition that got its own paragraph and that's it) -- ideally these two (for the purpose of donating) should be presented equally, rather than mentioning FFIS as an afterthought.